configure.ac

```diff
@@ -886,7 +886,13 @@ AC_CHECK_HEADER([proj_api.h],
 	)
 
 dnl Return the PROJ.4 version number
-AC_PROJ_VERSION([POSTGIS_PROJ_VERSION])
+if test ! -z "$PKG_CONFIG"; then
```

Comment:
There's a check for PKG_CONFIG earlier in the script where this could go much more sensibly. Then AC_PROJ_VERSION can just be conditional on POSTGIS_PROJ_VERSION not already being set.